docs/user_guide/vis/index.rst

```diff
@@ -216,13 +216,14 @@ The main visual components are called Units, Lines, and Annotations.
 Units
     *Units* is the term used for any geometric shape in the flowsheet that is connected by lines.
     The three types of units are IDAES unit models (such as a Flash, Mixer, or Splitter), inlets, or
-    outlets. Units can be moved by clicking and dragging them. If you double-click on a unit, it
+    outlets. Units can be moved by clicking and dragging them. If you right-click on a unit, it
     will rotate 90 degrees.
 
 Lines
     The lines connecting units, also called "arcs", can be manipulated by clicking and dragging.
     If you double-click on a line, you will create a new segment that can be used for routing the line
-    around objects.
+    around objects. You can elminate a segment by clicking on the dot that appears as you hover over
```

Comment:
Spelling error. Should be eliminate.

Reply:
my IDE tried to tell me, but I didn't listen! fixed